File: app/src/main/java/com/example/cobe/cryptonews/common/utils/ValidationUtils.java
package com.example.cobe.cryptonews.common.utils;

/**
 * Created by cobe on 03/04/2018.
 */

public class ValidationUtils {

    public static boolean isEmpty(String string) {
        return (string != null && string.isEmpty() && string.trim().isEmpty());
    }
}

File: app/src/main/java/com/example/cobe/cryptonews/common/utils/ValidationUtils.kt
package com.example.cobe.cryptonews.common.utils

/**
 * Created by cobe on 13/04/2018.
 */
class ValidationUtils {

    companion object {
        fun isEmpty(text: String): Boolean {
            return (text.isEmpty() && text.trim().isEmpty())
        }
    }
}